addons/mail/static/src/components/follower/follower.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.Follower" owl="1">
-        <div class="o_Follower">
+        <div class="o_Follower" t-on-click="props.onClick">
```

Comment:
where is that one coming from?

Reply:
`follower_list_menu` has `<Follower t-on-click/>`